openmetadata-docs/content/deployment/docker/index.md

```diff
@@ -67,6 +67,32 @@ Follow the instructions [here](https://docs.docker.com/compose/cli-command/#inst
   - Deploying with MySQL:  Download `docker-compose.yml` file from the above link.
   - Deploying with PostgreSQL: Download `docker-compose-postgres.yml` file from the above link.
 
+- Create the directory for host volumes 
+```commandline
+mkdir -p $PWD/docker-volume/db-data
+```
+
+- Update the docker-compose file which was downloaded from the above 
+  -  For MYSQL
+      ```commandline
+        version: "3.9"
+        services:
+          mysql:
+            ...
+            volumes:
+              - $PWD/docker-volume/db-data:/var/lib/mysql
+            ...
+      ```
+  -  For PostgreSQL
+        ```commandline
+          version: "3.9"
+          services:
+            mysql:
+              ...
+              volumes:
+                - $PWD/docker-volume/db-data:/var/lib/postgresql
+              ...
+        ```
```

Comment:
Is this required for Docker deployments ? I believe your PR handle's the volume mapping right ?

Reply:
Right